content/en/docs/setup/install/external-controlplane/index.md

```diff
@@ -57,6 +57,38 @@ need to modify the installation procedure to enable access. For example, the
 [multicluster configuration](#adding-clusters) could also be used to enable access
 to the API server.
 
+### Istio CA certificates
+
+Since Istio remote cluster will communicate with Istio external control plane cluster
+via TLS connections invoked by Istio mutating web hooks running in remote cluster,
+it is essential that the Istio mutating webhook in remote cluster trusts the
+Istio control plane injection service whose connection shall be encrypted by either
+the ingress gateway or the external control plane itself. This is especially important
+when your Istio certificates are self signed. Thus the external control plane
+root certificate (not to be confused with its private key) shall be available when set
+up your Istio component in remote cluster such as the mutating web hook.
+
+There are number of ways to obtain such certificate. If you are setting up a production environment,
+most likely your certificates come from Trusted Certificate Authority such as `VeriSign` or
+`SecureTrust`, you should already have the certificate in your possession, and you do not have to configure
+your Istio web hooks to trust additional certificates. However if you use self signed certificates either
```

Comment:
I don't think we want the self-signed passthrough approach to be a first class part of these instructions. I think we should just mention in a `{{< tip >}}` that you can do it and also in other tips (at the appropriate places in the instructions) show what to change in the config.

Reply:
ok. tip as well